packages/headless-styles/src/components/shared/_button.scss

```
@@ -0,0 +1,34 @@
+@use './states.scss' as *;
+
+@mixin pando-btn() {
```

Because this is now using SCSS, we don't need the pando name scope on all the classes for mixins.

```suggestion
@mixin base-btn() {
```